### ISSUE TITLE
Add Word Wrap option to events list in Live Data views

### ISSUE THREAD

**Issue by lnlp** [TheThingsNetwork/lorawan-stack#3718]
#### Summary
Add a Word Wrap option to events list in Live Data views. This is only needed for the Data Preview field.

#### Why do we need this?
To improve the UX and usability. To make visible all event summary data at a glance, without being required to horizontal scroll each event separately.

Events summaries are displayed on a single line but the lines can be long and often the right part is not visible.
One has to horizontally scroll each message separately to see the non-visible remainder of the line and this needs to be done with a very tiny scrollbar below the message. It is impossible to see all event summary data at a glance, especially when new events are added all the time.

#### What is missing? What do you want to see?
Missing is a Word Wrap option for the Data Preview field.
With word wrap enabled data in the Data Preview can continue on the next line (in same column to keep it tidy) so it can all be visible without falling off-screen.

#### How do you propose to implement this?
Add a checkbox somewhere in the top of Live Data windows to enable/disable the Word Wrap option.

**Comment by kschiffer** [comment]
Blocked on #3817

**Comment by KrishnaIyer** [comment]
Thanks for the issue and the long wait. This is a nice to have that we can do.
<img width="602" alt="Screenshot 2024-10-14 at 13 56 52" src="https://github.com/user-attachments/assets/f00e74d9-fb4f-49a0-a88d-7bf332d1bc17">

This is still relevant after the console redesign.

**Comment by KrishnaIyer** [comment]
This is tested and will be available in the `v3.33.0` release.
